ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -58,6 +58,32 @@ ColumnLayout {
         readonly property var sourceModel: root.controller.sourceModel
         readonly property bool isLoading: root.isUpdating || root.isFetching
 
+
+        function toggleSortByDate(value) {
+            console.info("bTest toggleSortByDate", value)
+            var tete = cmbTokenOrder.currentValue;
+            var orderByDateIndex =  cmbTokenOrder.indexOfValue(SortOrderComboBox.TokenOrderDateAdded);
+
+            var newModel = cmbTokenOrder.model;
+
+            newModel[ind].isDisabled = value;
+            cmbTokenOrder.model = newModel;
```

Comment:
`newModel` is just a reference, not a real new model, the assignment `cmbTokenOrder.model = newModel;` do nothing because it's the same reference.

Reply:
In order to trigger change signal you can do:
```
cmbTokenOrder.model[ind].isDisabled = value;
cmbTokenOrder.modelChanged()
```